packages/veritone-react-common/src/components/DataPicker/Breadcrumbs/index.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
```

Comment:
For the sake of simplicity / consistency, can we use material ui breadcrumbs, they also have extra functionality like collapsing.

Reply:
But I'm fine with this too.